_common/KeymanHosts.php

```diff
@@ -89,11 +90,14 @@ function __construct() {
         $site_suffix = '.local';
         $site_protocol = 'https://';
         break;
+      default:
+        die("tier is '$this->tier' which is invalid\n");
       }
 
       $this->blog_keyman_com = "https://blog.keyman.com";
       $this->donate_keyman_com = "https://donate.keyman.com";
       $this->translate_keyman_com = "https://translate.keyman.com";
+      $this->sentry_keyman_com = "https://sentry.keyman.com";
```

Comment:
TODO: As we dockerize more sites, the TIER_TEST and TIER_DEVELOPMENT will need to change
* Instead of `{$site_suffix} == .local `, map to port numbers?